eclass/gnome2.eclass

```diff
@@ -170,16 +162,8 @@ gnome2_src_install() {
 	rm -rf "${ED}/usr/share/applications/mimeinfo.cache" || die
 
 	# Delete all .la files
-	if has ${EAPI} 6; then
-		case "${GNOME2_LA_PUNT}" in
-			yes)    prune_libtool_files --modules;;
-			no)     ;;
-			*)      prune_libtool_files;;
-		esac
-	else
-		if [[ ${GNOME2_LA_PUNT} != 'no' ]]; then
-			find "${ED}" -name '*.la' -delete || die
-		fi
+	if [[ ${GNOME2_LA_PUNT} != no ]]; then
+		find "${ED}" -name '*.la' -delete || die
```

Comment:
Add `-type f` so it agrees with [PG 0303](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0303).

```suggestion
		find "${ED}" -type f -name '*.la' -delete || die
```